internal/core/block/block.go

```diff
@@ -30,7 +30,7 @@ import (
 // Schema is the IPLD schema type that represents a `Block`.
 var (
 	Schema          schema.Type
-	SchemaPrototype schema.TypedPrototype
+	SchemaPrototype ipld.NodePrototype
```

Comment:
question: I wonder where this `ipld` coming from. I don't see any import that ends with "ipld"

Reply:
`go-ipld-prime` is the package 🙂